static/js/admin.js

```diff
@@ -110,6 +127,15 @@ exports.setup_page = function () {
         error: failed_listing_streams
     });
 
+    // Populate emoji table
+    channel.get({
+        url: '/json/realm/emoji',
+        idempotent: true,
+        timeout: 10 * 1000,
+        success: populate_emoji,
+        error: failed_listing_emoji
+    });
```

Comment:
Yes, that's right, but I though this might need to be accessible through the API anyway, so I added it as an API endpoint. Should I just remove the query in the frontend or entirely remove this endpoint?